runtime/runtime/tests/test_async_calls.rs

```diff
@@ -6,6 +6,12 @@ use near_primitives::receipt::{ActionReceipt, ReceiptEnum};
 
 pub mod runtime_group_tools;
 
+/// Initial balance used in tests.
+pub const TESTING_INIT_BALANCE: u128 = 1_000_000_000 * NEAR_BASE;
```

Comment:
It's somewhat annoying to create this value multiple times in different places.

Reply:
I don't want to create dependencies between the crates just to share a constant.